### PR TITLE
docs: add pip-28 prevent node shotdowns

### PR DESCRIPTION
**Pull request by Ja7ad**

### CODE REVIEW

**Comment by kehiy** [comment]
@Ja7ad 
I think this is not a protocol change and its related to implementation. any implementation can have any features that won't change the protocol. So, it can be an issue on the https://github.com/pactus-project/pactus repo instead of an improvement proposal since it's only a protocol change.

If we put every software feature on PIPs we will have a huge number of PIPs then.

File: PIPs/pip-28.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+discussion-no:
+status: Draft
+type: Standards Track
+category: Core
+created: 06-07-2024
+---
+
+## Abstract
+
+PIP-28 proposes a protocol change to prevent validators from shutting down their nodes while actively
```
Comment (b00f) [inline PIPs/pip-28.md:14]: Try to keep abstract simple and in one line. Read Template for more help

**Comment by b00f** [comment]
Do we really need this feature? How much complexity we add to software to implement this feature?

**Comment by Ja7ad** [comment]
> Do we really need this feature? How much complexity we add to software to implement this feature?

This is does not have complexity, just handle in main function daemon.

**Review by b00f — APPROVED** [review]

**Pull request merged**